stl_text/ops/tokenizers/whitespace_tokenizer.py

```diff
@@ -1,44 +1,43 @@
 import torch.nn as nn
 from typing import Dict, List, Optional
+from collections import OrderedDict
 from iopath.common.file_io import PathManager
-
+from torchtext.experimental.vocab import vocab
 
 class WhitespaceTokenizer(nn.Module):
-    def __init__(self, vocab_path: Optional[str] = None, trainable=False, speed: int = 0):
+    def __init__(self, vocab_path: Optional[str] = None, trainable: bool = False):
         super(WhitespaceTokenizer, self).__init__()
         self.trainable = trainable
-        self.speed = speed  # mock a real tokenizer: slowing down tokenization speed
-
-        self.unknown = "unknown"
-        self.vocab: Dict[str, int] = {self.unknown: 0}
 
         # load vocab
-        path_manager = PathManager()
+        ordered_dict = OrderedDict()
         if vocab_path:
+            path_manager = PathManager()
             with path_manager.open(vocab_path, "r") as f:
                 for line in f.readlines():
                     token = line.split()[0]
-                    self.vocab[token] = len(self.vocab)
+                    ordered_dict[token] = 1
+ 
+        self.vocab = vocab(ordered_dict)
+        self.vocab = self.vocab.to_ivalue()
+
 
     def forward(self, text: str) -> List[int]:
         return self.numberize(self.tokenize(text))
 
     def tokenize(self, text: str) -> List[str]:
-        # slowing down tokenization speed
-        count = 0
-        for i in range(self.speed):
-            count += 1
         return text.split()
 
     def numberize(self, tokens: List[str]) -> List[int]:
-        token_ids: List[int] = []
-        for token in tokens:
-            if token in self.vocab:
-                token_ids.append(self.vocab[token])
-            else:
-                if self.trainable:
-                    token_ids.append(len(self.vocab))
-                    self.vocab[token] = len(self.vocab)
-                else:
-                    token_ids.append(self.vocab[self.unknown])
+        token_ids = self.vocab(tokens)
+
+        if self.trainable:
+            unk_id = self.vocab.__getitem__('<unk>')
```

Comment:
You should be able to use `unk_id = self.vocab(['<unk>'])` here. See the [doc](https://github.com/pytorch/text/blob/master/torchtext/experimental/vocab.py#L163).

Reply:
> Please double check the new whitespacetokenizer is jitable in the context.
> 
> ```
> tokenizer = WhitespaceTokenizer(vocab_path)
> jit_tokenizer = torch.jit.script(tokenizer)
> ```

Yes, this is validated via doc_classification workflow (it seems to not working for non-jitable WhitespaceTokenizer)